modules/administration/pages/virtualization.adoc

```diff
@@ -315,6 +315,10 @@ Apart from being able to serve as a host for {kvm} or {xen} guests, which are ma
 ** Execute the bootstrap script to register the {vmhost}.
 +
 
+[IMPORTANT]
+====
+After bootstrap, Salt minions need to be restarted in order to make virtualization fully work.
```

Comment:
Would rather write:

    in order to get instant virtual machines changes reflected in {productname}

Reply:
see my recent sugggestion at the end of this thread.